atlasdb-workload-server-api/src/main/java/com/palantir/atlasdb/workload/transaction/ReadTransactionAction.java

```diff
@@ -23,7 +23,7 @@
 import org.immutables.value.Value;
 
 @Value.Immutable(builder = false)
-public interface ReadTransactionAction extends TransactionAction {
+public interface ReadTransactionAction extends SingleCellTransactionAction {
```

Comment:
Oh, might be worth calling this ReadSingleCellTransactionAction to differentiate it a bit better from the row column range stuff, but can be done in a separate PR if that's easier

Reply:
I'll do this at the end of the branch to avoid nasty merge conflicts, but makes sense.